src/components/Parser/VastURLParser.js

```diff
@@ -79,7 +81,21 @@ class VastURLParser {
             params[decodedKey] = decodedValue;
           }
         } else {
-          params[decodedKey] = decodeURIComponent(value);
+          const decodedParam = vastAdTagParameters.filter(parameter => {
```

Comment:
This could be simplified:
```
if (decodedKey) {
  const decodedParam = vastAdTagParameters.filter(parameter => 
    parameter.name === decodedKey || 
    (parameter.aliases && parameter.aliases.includes(decodedKey))
  );
  if (decodedParam.length > 0) {
    params[decodedParam[0].name] = decodeURIComponent(value);
  } else {
    params[decodedKey] = decodeURIComponent(value);
  }
}
```